dtfs-central-api/api-tests/v1/utilisation-reports/get-utilisation-reports.api-test.ts

```diff
@@ -0,0 +1,62 @@
+import { Response } from 'supertest';
+import wipeDB from '../../wipeDB';
+import app from '../../../src/createApp';
+import db from '../../../src/drivers/db-client';
+import { DB_COLLECTIONS } from '../../../src/constants/db-collections';
+import { MOCK_UTILISATION_REPORT } from '../../mocks/utilisation-reports/utilisation-reports';
+import { UtilisationReport } from '../../../src/types/db-models/utilisation-reports';
+import createApi from '../../api';
+
+const api = createApi(app);
+
+const getUrl = (bankId: string) => `/v1/bank/${bankId}/utilisation-reports`;
+
+interface CustomResponse extends Response {
+  body:
+    | {
+        errors: {
+          msg: string;
+        }[];
+      }
+    | ({ errors?: undefined } & UtilisationReport);
```

Comment:
Just trying to read this type - if we have a UtilisationReport would we not expect errors to not exist? What makes it not:

```
{
        errors: {
          msg: string;
        }[];
      }
    | UtilisationReport
    ```

Reply:
Is it just so errors always has a chance of existing when looking at the object in this file?

Reply:
Yeah exactly, but if there is a utilisation report then errors will be undefined.

Reply:
Cool, I think the ? is obsolete then if its defnitely undefined

Reply:
Yeah, you're right - it is obsolete in the case. 

This is a pattern I follow for these type of unions so that you don't need to specify the value at all when manually constructing (i.e. you are not forced to add `errors: undefined`), but in this case where it just represents the response then it doesn't make any difference.